tests/Main/ReflectionTests.fs

```diff
@@ -364,7 +364,10 @@ let reflectionTests = [
             let types =
                 info.GetFields()
                 |> Array.map (fun field -> field.PropertyType)
-            info.Tag, info.Name, types, values)
+            let names =
```

Comment:
A bit duplicative, but I figured it doesn't matter in a unit test, not worth trying to change the test structure for it.